### PR TITLE
Add tag for scodix textures

### PR DESCRIPTION
**Pull request by zsapozhnikov**
Fixes https://tfs.farheap.com/ONP/overnightprints.com/_workitems/edit/31755?fullScreen=false

### CODE REVIEW

File: src/InkRouter/Models/Side.php [hunk 1]
```diff
@@ -234,6 +251,44 @@ public function getPrintAssets()
...
+        return $this->variableUv;
+    }
+
+    /**
+     * @param bool $variableUv
+     * @return InkRouter_Models_Side
+     */
+    public function setVariableUv($variableUv)
+    {
+        $this->variableUv = $variableUv;
```
Comment (maria-p) [inline src/InkRouter/Models/Side.php]: i don't think it's a good idea to make it inconsistent with current file specs in xml.
why would we need bool? if url is there, then order has variable uv. if they really want it, why don't we have it for spot?
also, we must have file hash to handle order updates.
please contact BAs/profit team again.

Comment (zsapozhnikov) [inline src/InkRouter/Models/Side.php]: ping @RinatKhanbekov

**Comment by RinatKhanbekov** [comment]
Confirm that the boolean flag can be removed, ProFIT will check the presence of Scodix coating on the side of given order item by looking at the variable_uv_file_url parameter on the XML.
Sorry for the inconvenience.

**Comment by richardfullmer** [comment]
Agreed.  Need hashes.  Is this what the ProFIT team implemented?  Why is the `UV` uppercased on one of the XML nodes?

**Comment by RinatKhanbekov** [comment]
Currently ProFIT does not check hashes for any types of images ONP sends there (CMYK or SpotUV), same is for Scodix files - ProFIT is not going to validate hash for them for now.

**Pull request merged**